nav2_follow_path_controller/include/nav2_follow_path_controller/nav2_follow_path_controller.hpp

```diff
@@ -64,25 +69,26 @@ class DwbController : public nav2_util::LifecycleNode
   bool isGoalReached();
   bool getRobotPose(nav_2d_msgs::msg::Pose2DStamped & pose2d);
 
-  // The DWBController contains a costmap node
+  // The local controller needs a costmap node
   std::shared_ptr<nav2_costmap_2d::Costmap2DROS> costmap_ros_;
   std::unique_ptr<std::thread> costmap_thread_;
 
   // Publishers and subscribers
   std::shared_ptr<nav_2d_utils::OdomSubscriber> odom_sub_;
   rclcpp_lifecycle::LifecyclePublisher<geometry_msgs::msg::Twist>::SharedPtr vel_publisher_;
 
-  // The local planner
-  std::unique_ptr<dwb_core::DWBLocalPlanner> planner_;
+  // Local Planner Plugin
+  pluginlib::ClassLoader<nav2_local_planner_interface::LocalPlannerInterface> blp_loader_;
```

Comment:
`nav2_core::LocalPlanner`, it itself is an interface, it doesn't need to be named an interface

Reply:
I agree.